test/config/generis.conf.php

```diff
@@ -39,7 +39,9 @@
 define('GENERIS_SESSION_NAME', 'test');
 
 # paths
-define('ROOT_PATH', '');
+if (!defined('ROOT_PATH')) {
+    define('ROOT_PATH', '');
```

Comment:
Are there entrypoints to this file other than `test/bootstrap.php`? 🤔

Reply:
I do not know. I'm afraid that it may be used by other extensions so I left definition of this constant there.

Reply:
To guarantee proper test configuration the bootstrap file always need to be used. All these cases should be unveiled and fix if there's any rather than being suppressed IMO.